arrocke/gloss-translation
# fix: notes editor update issue

## What has changed
- Fixed the bug described in #401
   - The method of updating the editor's content has switched from using state to using refs.


## Connected Issues
closes #401 


## QA Steps
1. Find two words without notes.
2. Open the notes tab for the first word. Enter any content into one of the notes editors.
3. Switch focus to the second word.
4. Confirm that the editor is empty.
5. Now enter content into the editor of the second word.
6. Switch the translation language to English. Confirm that the editor is empty (or contains only the english notes)

## Post-Deployment
